modules/tests/jvm/src/main/scala/gsp/math/geom/jts/demo/JtsDemo.scala

```diff
@@ -4,38 +4,40 @@
 package lucuma.core.math.geom.jts
 package demo
 
-import lucuma.core.math.{Angle, Offset}
+import lucuma.core.math.{ Angle, Offset }
+import lucuma.core.math.geom.GmosOiwfsProbeArm
+import lucuma.core.math.geom.GmosScienceAreaGeometry
 import lucuma.core.math.geom.ShapeExpression
 import lucuma.core.math.geom.jts.jvm.syntax.awt._
 import lucuma.core.math.geom.jts.interpreter._
 import lucuma.core.math.geom.syntax.shapeexpression._
 import lucuma.core.math.syntax.int._
 
-import java.awt.{List => _, _}
+import java.awt.{ List => _, _ }
 import java.awt.event._
 
 import scala.jdk.CollectionConverters._
 
 /**
- * Throwaway demo code to visualize a shape created using `ShapeExpression`s.
- */
+  * Throwaway demo code to visualize a shape created using `ShapeExpression`s.
+  */
 object JtsDemo extends Frame("JTS Demo") {
 
-  val posAngle: Angle         =
+  val posAngle: Angle =
     145.deg
 
   val guideStarOffset: Offset =
     Offset(170543999.µas.p, -24177003.µas.q)
 
-  val offsetPos: Offset       =
+  val offsetPos: Offset =
     Offset(60.arcsec.p, 60.arcsec.q)
 
   // TODO: should come from the FPUnit enum in core
-  val ifuOffset: Offset       =
+  val ifuOffset: Offset =
     Offset.Zero
 
   // TODO: when we move this core there should be an enum
-  val sideLooking: Boolean    =
+  val sideLooking: Boolean =
     true
```

Comment:
This code does differ a bit in `gsp-core`, referencing a `PortDisposition` enumeration here for example.  Will those changes get picked up as part of this process?

Reply:
We had two versions for GMOS geometries. This PR will unify them using the one form gsp core which has a proper use of enumerations

Reply:
Yes, @cquiroz was migrating this. Thank you @cquiroz !